src/status_im/tests/contract_test/utils.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.tests.contract-test.utils
+  (:require
+   [status-im.common.json-rpc.events :as rpc-events]
+   [utils.number]))
+
+(defn check-response [validator response callback]
+  (let [test-checker (js/Promise.
+                      (fn [res reject]
+                        (if (validator response)
+                          (res "test passed")
+                          (reject "test failed"))))]
+    (.then test-checker
+           (fn [value] (callback value))
+           (fn [reason] (println "Test failed due to:" reason)))))
+
+(defn call-rpc-endpoint
+  [{:keys [rpc-endpoint
```

Comment:
From the JSON RPC spec, `rpc-endpoint` would be preferably named `method`, just like the `:json-rpc/call` effect.